examples/simple-nginx.md

```diff
@@ -37,7 +37,13 @@ to do this run:
 kubectl expose rc my-nginx --port=80 --type=LoadBalancer
 ```
 
-This should print the service that has been created, and map an external IP address to the service.
+This should print the service that has been created, and map an external IP address to the service. Where to find this external IP address will depend on the environment you run in. For instance, for GCE you can run the following to find the newly created forwarding rule that serves as the external IP address to the service.
```

Comment:
I see. I think what you're suggesting is instead to suggest to users to run

kubectl get services

and look for the external IP address displayed (and then we wouldn't have to mention GCE at all here). Right?